src/components/app/TabBar.js

```diff
@@ -51,6 +52,21 @@ class TabBar extends PureComponent {
   }
 }
 
+type Props = {
+  className?: string,
+  tabs: Tab[],
+  selectedTabName: string,
+  tabOrder: number[],
+  onSelectTab: string => mixed,
+  onChangeTabOrder: (number[]) => any,
+  +_mouseDownListener?: () => mixed,
```

Comment:
I guess `+` wasn't needed. It snuck in as I was trying to figure out the `(this: any).` stuff with `_mouseDownListener`.